### DISCUSSION TITLE
Push from workflow not doesn't trigger on.push.tags workflow

### DISCUSSION THREAD

**Discussion by robtayl0r** [community/community#27194] [Actions]
<p>I have a workflow that triggers on push branches:</p>
<pre><code class="lang-yaml">    steps:
      - uses: actions/checkout@v2
      - uses: &lt;some simple git tag action&gt;
      - name: Push Version Update
        run: git push origin --follow-tags
</code></pre>
<p>Now the job is successful, the push tag step is successful, and I see the tag in the remote repository. However, a subsequent workflow to publish does not get triggered:</p>
<pre><code class="lang-yaml">on:
  push:
    tags: ['v*']

</code></pre>
<p>If I manually create a tag and push it to the remote, the on.tags workflow will trigger.</p>
<p>I believe this is related to some internal Actions mechanism as to avoid workflow infinite loops. I realize I could be butchering the adoption of Actions with this approach. any ideas or suggestions as to how to properly adopt this type of workflow using Actions, where one workflow creates a tag and pushes it back to its repository, which can then be observed and trigger another flow?</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>By default, <a href="https://github.com/actions/checkout"><code>actions/checkout</code></a> configures <code>GITHUB_TOKEN</code> for authentication. The problem is that events caused by <code>GITHUB_TOKEN</code> cannot trigger new workflows as described in <a href="https://docs.github.com/en/free-pro-team@latest/actions/reference/authentication-in-a-workflow#using-the-github_token-in-a-workflow">Using the <code>GITHUB_TOKEN</code> in a workflow</a>:</p>
<blockquote>
<p>When you use the repository’s <code>GITHUB_TOKEN</code> to perform tasks on behalf of the GitHub Actions app, events triggered by the <code>GITHUB_TOKEN</code> will not create a new workflow run.</p>
</blockquote>
<p>If you want the push to create new workflow runs you need to <a href="https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/creating-a-personal-access-token">use a PAT instead</a>. After adding it as a <a href="https://docs.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets">secret</a> you can do something like this to use it instead of <code>GITHUB_TOKEN</code>:</p>
<pre><code class="lang-yaml">- uses: actions/checkout@v2
  with:
    token: ${{ secrets.PAT }}
</code></pre>

**Reply by sebthom** [reply]
This doesn't seem to work (anymore?). However it works when using a deploy key instead of a PAT.

**Reply by I-Want-ToBelieve** [reply]
This doesn't seem to work! 
I will try  deploy key

**Comment by robtayl0r** [comment]
<p>Thanks for the reply. It seems inappropriate for a CI job to use a PAT. If I leave the organization, the workflow will just start failing.</p>
<p>I guess I’ll try to figure out another way to trigger or just combine the workflows.</p>
